tests/test_requests/test_allowed_request_types_link_and_service.py

```diff
@@ -227,8 +227,8 @@ def test_ui_serialization(
             "description": "Request deletion of published record",
             "name": "Delete record",
             "dangerous": True,
-            "editable": False,
-            "has_form": False,
+            "editable": True, # idk why this changed to True, could be because of form 
```

Comment:
Please try to keep this to False - if False, saving the request would automatically submit it, if True, after the request is created, it is in open state and has to be submitted manually. The "editable" is meant for larger forms, where you need to fill in a lot of fields and would like to save it in between